src/content/deployment/android.md

```diff
@@ -92,8 +92,7 @@ To find out the latest version, visit [Google Maven][].
 +<style name="NormalTheme" parent="Theme.MaterialComponents.DayNight.NoActionBar">
 ```
 
-<a id="signing-the-app"></a>
-## Sign the app
+## Sign the app {:#signing-the-app}
```

Comment:
Please keep the separate anchor here so they both continue to work. There's old links to both headers from outside the site.

Reply:
In the site checks out as OK.

Reply:
Yes in the site is fine, but these extra anchors are here for outside the side, such as from old framework and tool versions, other sites, etc.